rpcs3/Emu/Cell/lv2/sys_ss.cpp

```diff
@@ -2,66 +2,106 @@
 #include "sys_ss.h"
 
 #include "sys_process.h"
+#include "Emu/IdManager.h"
 #include "Emu/Cell/PPUThread.h"
 
 
 #ifdef _WIN32
 #include <Windows.h>
 #include <wincrypt.h>
+#endif
 
-const HCRYPTPROV s_crypto_provider = []() -> HCRYPTPROV
```

Comment:
Still didn't get why you moved this code, it's not like it needs to be reset between restarts.